toolset/setup/linux/scaffolding.py

```diff
@@ -0,0 +1,236 @@
+import sys
+import os
+import subprocess
+from shutil import copytree
+
+class Scaffolding:
+  def __init__(self):
+    print """
+-------------------------------------------------------------------------------
+    This wizard is intended to help build the scaffolding required for a new 
+    test to be benchmarked.
+
+    From here, you will be prompted for values related to the test you
+    wish to add.
+-------------------------------------------------------------------------------"""
+
+    # try:
+    self.__gather_display_name()
+    self.__gather_language()
+    self.__gather_approach()
+    self.__gather_classification()
+    self.__gather_orm()
+    self.__gather_webserver()
+    self.__gather_versus()
+
+    self.__confirm_values()
+    # except:
+      # print ""
+
+  def __gather_display_name(self):
+    print """
+  The name of your test as you wish it to be displayed on the results page.
+    """
+    self.__prompt_display_name()
+    while not self.display_name:
+      self.__prompt_display_name()
+    self.name = self.display_name.lower()
+
+  def __prompt_display_name(self):
+    self.display_name = raw_input("Name: ").strip()
+
+  def __gather_language(self):
+    print """
+  The language in which your test implementation is written.
+
+  Example: Java, Go, PHP
+    """
+    self.language = None
+    while not self.language:
+      self.__prompt_language()
+
+  def __prompt_language(self):
+    self.language = raw_input("Language: ").strip()
+    return self.language
+
+  def __gather_approach(self):
+    print """
+  The approach of your test implementation.
+
+  Realistic: Uses the framework with most out-of-the-box functionality enabled.
+             We consider this realistic because most applications built with 
+             the framework will leave these features enabled.
+  Stripped:  Removes or outright avoids implementing features that are
```

Comment:
Personally, I wouldn't mind if we never got another Stripped implementation.  So I would assume Realtistic here and not even ask.  But if you do want to ask, I think it would be fair to point out that Stripped implementations are hidden by default on the official TFB website.

Reply:
Good point; I'll add that.